src/pages/home/subscription/subscription.styled.js

```diff
@@ -67,20 +67,14 @@ export const JoinButton = styled(Button)`
 `;
 
 export const EmailInput = styled(Input.Search)`
-  height: 45px;
   margin-top: 2rem;
   border-radius: 6px;
   max-width: 350px;
 
-  span {
-    font-size: 16px;
-  }
-
+  &,
+  .ant-btn,
   .ant-input-affix-wrapper {
-    height: 45px;
-  }
-
-  button.ant-btn {
+    font-size: 16px;
```

Comment:
We can merge the same stylings for the input wrapper, `.ant-btn` and `.ant-input-affix-wrapper` as above.